robot/impl/local_robot.go

```diff
@@ -416,8 +416,12 @@ func newWithResources(
 
 	var ftdcWorker *ftdc.FTDC
 	if rOpts.enableFTDC {
+		partID := "local-config"
+		if cfg.Cloud != nil {
```

Comment:
bug fix

Reply:
What bug was this? Just looks like starting FTDC within a `~/.viam/[part-id|"local-config"]` as a new feature.

Reply:
Oh nvm I see your other comment.